submissions/sinchUK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Popup</title>
+</head>
+<body>
+    <header class="menu">
+        <nav>
+            <ul class="menu-items">
+                <li class="menu-item">
+                    <a href="#">
+                        Mail
+                    </a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">
+                        Images
+                    </a>
+                </li>
+                <li class="menu-item">
+                    <div class="menu-item__inner">
+                        <input class="menu-item__input" type="checkbox"/>
+                        <img class="menu-item-img" src="./icons/popup-button.png" alt="submenu-icon"/>
```

Comment:
Add `cursor: pointer` to all interactive elements.

Reply:
![Screenshot 2022-10-20 134916](https://user-images.githubusercontent.com/17921586/196928846-44807497-83c3-442f-a357-1daccf83df16.png)
I delete the inner text in "alt" attribute from icons. Thank you for good article.